### ISSUE TITLE
As a Pipes developer or user, I'd like to know more details about the Pipes version I'm using

### ISSUE THREAD

**Issue by josvanroosmalen** [marklogic-community/pipes#133]
The version number show in the start screen and console (e.g. `1.0.1`) should be determined dynamic.  "What is new" should be removed from the flash screen, and should be replaced with a link referring to a website. 

Release builds should only be build on tagged versions ( `git checkout R1.0.0`), then:

``version=`git describe --tags` ``

In case of a nightly build use a short commit id:

``version=`git rev-parse --verify --short HEAD` ``

You can add this to the build script.

**Comment by sashamitrovich** [comment]
@josvanroosmalen , again - can you rephrase this ticket to explain what problem is it addressing?

**Comment by josvanroosmalen** [comment]
This solves the problem that we always know: (1) what the source (commit id) is for this build and (2) for release builds the version numbers are always correct.

**Comment by sashamitrovich** [comment]
> @josvanroosmalen , again - can you rephrase this ticket to explain what problem is it addressing?

Absolutely - was just clarifying that I understand what we're targeting with it.

**Comment by sashamitrovich** [comment]
Information we want to expose:
- Is this a release build or a developer build?
- What is the build identifier (can use commit hashes here)?

**Comment by sashamitrovich** [comment]
Version information now created automatically by the build script and:
- printed on Pipes startup in the terminal
- printed in the splash screen in the UI
- part of the JAR file name

<img width="635" alt="Screenshot 2020-04-21 at 15 00 25" src="https://user-images.githubusercontent.com/15609655/79868943-f71ce380-83e0-11ea-8549-db10d90312dc.png">

<img width="800" alt="Screenshot 2020-04-21 at 15 01 34" src="https://user-images.githubusercontent.com/15609655/79869000-0ac84a00-83e1-11ea-8b28-01b8338f04f9.png">

<img width="790" alt="Screenshot 2020-04-21 at 15 01 46" src="https://user-images.githubusercontent.com/15609655/79869010-0ef46780-83e1-11ea-935c-f7e42d688046.png">
